genie-core/src/main/java/com/netflix/genie/core/jobs/JobConstants.java

```diff
@@ -135,6 +135,11 @@
     public static final String GENIE_KILL_REASON_FILE_NAME = "genie/kill-reason";
 
     /**
+     * File created by Genie with details and trace for a job that failed to initialize.
+     **/
+    public static final String GENIE_INIT_FAILURE_MESSAGE_FILE_NAME = "init-failure-details";
```

Comment:
I can add the extension (most other files don't have one, so i was just going with convention).
Same for dashes in the name. (What's the concern with those?)